CONTRIBUTING.md

```diff
@@ -33,26 +35,32 @@ A single resource has the following structure, where all attributes are required
 You can now go ahead and make changes to this file, using one of the following option.
 
 ### Using the UI
-Navigate to the `contnet` directory and click on the `resources.json` file. Here you can click on the pencil icon to edit the file. 
+
+Navigate to the `contnet` directory and click on the `resources.json` file. Here you can click on the pencil icon to edit the file.
 
 ![grafik](https://user-images.githubusercontent.com/28757404/142865728-4d26e83a-8e60-4c33-809b-02198fa0506f.png)
 
 You can now make your changes right inside the browser. Once you are done commit your changes by using the dedicated section below the displayed file content.
 
 ![grafik](https://user-images.githubusercontent.com/28757404/142865981-a8be3047-26ce-45dc-82b6-1318887dadde.png)
 
-
 ### Make changes locally
-Once you have forked the repository, go ahead and clone it locally. 
+
+Once you have forked the repository, go ahead and clone it locally.
 You need to install or update [node.js](https://nodejs.org/en/)
 Navigate into the cloned directory and run the following command inside the terminal to install the project dependencies.
+
 ```bash
 npm install
 ```
 
 First of all, checkout to a new branch to make your changes in. Now you can edit the `/content/resources.json` file. If you want to preview your changes locally you can simply run the following command:
+
 ```bash
 npm start
 ```
 
-Once you have all your changes made don't forget to commit them to the new branch and push it to github. Back in the github web-interface you can now create a Pull Request to merge your changes to the main branch of this repository.
+Once you have all your changes made don't forget to commit them to the new branch and push it to github. Back in the github web-interface, navigate to your repository fork where you can now create a Pull Request to merge your changes to the main branch of this repository.
```

Comment:
nitpick(blocking): Once you have all your changes made, don't forget to commit them to the new branch and push it to GitHub. Back in the GitHub web interface, navigate to your repository fork, where you can now create a Pull Request to merge your changes to the main branch of this repository.